packs/rabbitmq/actions/publish_message.py

```diff
@@ -0,0 +1,13 @@
+from kombu import BrokerConnection, Exchange, Queue
```

Comment:
This pack is built around pika. Using kombu probably works as it is a core stackstorm dependency but it is perhaps better to use pika.

Something like https://www.rabbitmq.com/tutorials/tutorial-one-python.html

Pika generally makes for simpler APIs than kombu. In core stackstorm also we are considering a move towards pika.